sphinx/index.md

```diff
@@ -0,0 +1,158 @@
+# OpenOA Documentation
+
+## Installation
```

Comment:
@RHammond2 I just looked at the compiled documentation, and think it would make more sense to start with the high-level software overview before the installation steps, to introduce people to the purpose of the software first. The order could be Software overview (intro part), installation, then analysis methods, PlantData Schema... What do you think about that order?

Reply:
I think that makes a lot of sense, I'll make that change and do the 3.1 release if there's nothing else.

Reply:
That sounds good to me! I don't think there's anything else, at least I didn't have any more comments.